docs/src/man/basics.md

```diff
@@ -0,0 +1,1417 @@
+ 1000 │   999     27  male        2  own      moderate         moderate        ⋯
+                                                               4 columns omitted
+```
+
+## Taking a subset of `german` data frame
+
+### Indexing Syntax
+
+Specific subsets of a data frame can be extracted using the indexing syntax, similar to matrices.
+In the [Indexing](https://dataframes.juliadata.org/stable/lib/indexing/#Indexing) section of the
+manual you can find all details about the available options. Here we highlight the basic ones.
+
+The colon `:` indicates that all items (rows or columns depending on its position) should be retained:
+
+```jldoctest dataframe
+julia> german[1:5, :]
+5×10 DataFrame
+ Row │ id     Age    Sex     Job    Housing  Saving accounts  Checking account ⋯
+     │ Int64  Int64  String  Int64  String   String           String           ⋯
+─────┼──────────────────────────────────────────────────────────────────────────
+   1 │     0     67  male        2  own      NA               little           ⋯
+   2 │     1     22  female      2  own      little           moderate
+   3 │     2     49  male        1  own      little           NA
+   4 │     3     45  male        2  free     little           little
+   5 │     4     53  male        2  free     little           little           ⋯
+                                                               3 columns omitted
+
+julia> german[[1, 6, 15], :]
+3×10 DataFrame
+ Row │ id     Age    Sex     Job    Housing  Saving accounts  Checking account ⋯
+     │ Int64  Int64  String  Int64  String   String           String           ⋯
+─────┼──────────────────────────────────────────────────────────────────────────
+   1 │     0     67  male        2  own      NA               little           ⋯
+   2 │     5     35  male        1  free     NA               NA
+   3 │    14     28  female      2  rent     little           little
+                                                               3 columns omitted
+
+julia> german[:, [:Age, :Sex]]
+1000×2 DataFrame
+  Row │ Age    Sex
+      │ Int64  String
+──────┼───────────────
+    1 │    67  male
+    2 │    22  female
+    3 │    49  male
+    4 │    45  male
+    5 │    53  male
+    6 │    35  male
+    7 │    53  male
+    8 │    35  male
+  ⋮   │   ⋮      ⋮
+  994 │    30  male
+  995 │    50  male
+  996 │    31  female
+  997 │    40  male
+  998 │    38  male
+  999 │    23  male
+ 1000 │    27  male
+      985 rows omitted
+```
+
+In this example we have created a `DataFrame` using `:Sex` and `:Age` column for the first five Rows:
+
+```jldoctest dataframe
+julia> german[1:5, [:Sex, :Age]]
+5×2 DataFrame
+ Row │ Sex     Age
+     │ String  Int64
+─────┼───────────────
+   1 │ male       67
+   2 │ female     22
+   3 │ male       49
+   4 │ male       45
+   5 │ male       53
+```
+
+Pay attention that `german[!, [:Sex]]` and `german[:, [:Sex]]` return a data frame object,
```

Comment:
why do you use `!` here for row selector? Also you have not explained yet what `german[!, [:Sex]]` means.

Reply:
I have explained about `[:Sex]` below the code blocks.

Reply:
This is exactly the point - the text does not follow the principle that **first** something is explained, and **next** it is used.